src/js/help/renderGallery.js

```diff
@@ -1,42 +1,88 @@
-'use strict';
-//DOM
-const gallery = document.querySelector('.gallery');
-
-const createItem = item => {
-  const { comments, downloads, likes, largeImageURL, tags, previewURL, views } =
-    item;
-  let itemMarkup = `<a class="gallery__item" href="${largeImageURL}" toClear>
-  <img class="gallery__image" src="${previewURL}" alt="${tags}" loading="lazy"/>
-  <div class="gallery__image-info">
-    <div class="gallery__image-info-item">
-        <p><b>Likes</b></p>
-        <p class="gallery__image-info-display">${likes}</p>
-    </div>
-        <div class="gallery__image-info-item">
-        <p><b>Views</b></p>
-        <p class="gallery__image-info-display">${views}</p>
-    </div>
-    <div class="gallery__image-info-item">
-        <p><b>Comments</b></p>
-        <p class="gallery__image-info-display">${comments}</p>
+import axios from 'axios';
```

Comment:
trochę niepotrzebnie  robiłaś wszystkie fetche - to jest rola Piotrka Koczkodaj'a - bardziej chodziło o stworzenie modułu który tylko dostaje JAKIEŚ dane - np. plik testowy i wypluwa galerie.